### ISSUE TITLE
Invalid ARIA attribute and value

### ISSUE THREAD

**Issue by vladimirivanoviliev** [telerik/kendo-ui-core#2559]
Currently the NumericTextBox have the following (aria) validation errors:

- Invalid attribute: `aria-readonly` should not be set to `role=spinbutton`
- Invalid attribute value: `aria-valuenow` is empty when no value is set. It should be set to `null`

Related to #2555
